modules/ROOT/pages/create-instance-task-mule.adoc

```diff
@@ -6,18 +6,14 @@ endif::[]
 
 include::partial$task-add-api-instance.adoc[tags=intro1;mule-intro;intro2]
 
+[[add-api]]
 == Add a New API
 
 . Navigate to *Anypoint Platform* > *API Manager*.
 . In *API Administration*, click *Add API* and select *Add new API*.
 . Select *Mule Gateway*.
 . Select the *Proxy type* from the following options:
 +
-Mule Gateway:: Select Mule Gateway if you have APIs on Mule Runtime that need an API gateway to manage, observe,
```

Comment:
I'm not sure why this is being deleted

Reply:
I think we're deleting it because with the Cloudhub update this information contradicts/needs to be changed.